people/jerrodbates.json

```diff
@@ -0,0 +1,32 @@
+{
+  "firstname": "Jerrod",
+  "lastname": "Bates",
```

Comment:
Please re-insert the handle field but leave it empty, without this it will produce an "undefined" message on the website